ontology/current-version/RiC-O_1.1.rdf

```diff
@@ -1921,6 +1921,23 @@
          </rdf:Description>
       </skos:changeNote>
    </owl:ObjectProperty>
+   <!-- https://www.ica.org/standards/RiC/ontology#dateAssociatedWithRelation -->
```

Comment:
Or maybe 'isDateAssociatedWithRelation', like for other date properties of the kind (I know  I'm splitting hairs...)